packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -3512,7 +3551,7 @@ class _CollapsedSelectionBoundary extends _TextBoundary {
   @override
   TextPosition getTrailingTextBoundaryAt(TextPosition position) {
     return isForward
-      ? innerTextBoundary.getTrailingTextBoundaryAt(position)
+        ? innerTextBoundary.getTrailingTextBoundaryAt(position)
```

Comment:
The change here doesn't look intended?

Reply:
it is actually.. to correct a previous unintended formatting change . this will not affect  the master.